### DISCUSSION TITLE
Draw outside of bounds of repositioned/rotated layer

### DISCUSSION THREAD

**Discussion by jsllllc** [keiwando/toonsquid#466] [Q&A]
Repositioning a drawing (not selecting pixels) creates an offset / rotated rectangular border at the original frame-size that cannot be drawn past. I would like to be able to draw past this. 

For small / static images, the solution would be to enable the toggle to select and reposition the pixels. However, this doesn't work in situations involving motion-keyframed objects. For example, if my motion-keyframed layer ultimately reveals undesired flat cropped sides, I can't draw outside the border to add the missing portion post-hoc. The only fix I've found requires creating a new layer with the missing portion and trying to get the motion to match up.

This might be too fundamental to how layer selection works to request as a new feature, so let me know if there is a solution I am missing. Thanks everyone.

**Comment by Karczel** [comment]
you may do this with camera layer + rounded diamond icon in the bottom left and set the position of the camera by resize move or rotate the camera layer in the canvas space, with different frame, the app will do the adding keyframe thing for you
I think you can adjust the move pace in the curves section (the next button to this)
![IMG_1B31C23B8B4F-1](https://github.com/user-attachments/assets/b0b9464a-c351-4c05-8896-a626ad5665a0)

**Comment by Cat-Butler** [comment]
Vector brushes are not constrained by layer size.
Alternatively, symbol layers can be a different size to the main timeline canvas, so you can insert a larger image (good for e.g. panning backgrounds).

Otherwise, yes, pixel layers act very much like the cels in old-fashioned cel animation. You can move them, and/or re-draw the content in the canvas (cel) area, but it is still  a standard-sized cel.
